### PR TITLE
Dev vs

### PR DESCRIPTION
**Pull request by VincentStimper**
* Changed adjusting weights and log prob before update in prioritized replay
* New default parameters for std in internal coordinate transform
* Improved treatment of `NaN`s when evaluating Ramachandran plot
* Refactored HMC class name

### CODE REVIEW

File: examples/aldp.py [hunk 1]
```diff
@@ -342,13 +359,50 @@ def initial_sampler():
...
                                          min_sample_length=rb_config['min_length'] * batch_size,
                                          initial_sampler=initial_sampler, device=str(device))
-        buffer_sample = None
         # Set target distribution
         def ais_target_log_prob(x):
             return 2 * model.target_distribution.log_prob(x) - model.flow.log_prob(x)
         model.annealed_importance_sampler.target_log_prob = ais_target_log_prob
 else:
     use_rb = False
+    if filter_chirality_train:
```
Comment (VincentStimper) [inline examples/aldp.py:368]: I had to add these to be able to filter for chirality during training also for the baseline models.

File: examples/aldp.py [hunk 2]
```diff
@@ -437,11 +486,13 @@ def ais_target_log_prob(x):
             x, log_w, log_q_old, indices = x.to(device), log_w.to(device), \
                                            log_q_old.to(device), indices.to(device)
             log_q_x = model.flow.log_prob(x)
-            # adjustment to account for change to theta since sample was last added/adjusted
-            w_adjust = torch.exp(log_q_old - log_q_x).detach()
-            w_adjust = torch.clip(w_adjust, max=rb_config['max_adjust_w_clip'])
-            # manually calculate the new form of the loss
+            # Adjustment to account for change to theta since sample was last added/adjusted
```
Comment (VincentStimper) [inline examples/aldp.py:489]: Please check if this is implemented correctly, @lollcat.

Comment (lollcat) [reply examples/aldp.py:489]: I ran a test on the GMM problem where I removed the w_adjust clipping and learning was still stable. So it may be worth adding an option here not to clip.

**Review by lollcat — APPROVED** [review]

File: examples/aldp.py [hunk 3]
```diff
@@ -437,11 +486,13 @@ def ais_target_log_prob(x):
...
+            log_w_adjust = log_q_old - log_q_x.detach()
+            w_adjust = torch.clip(torch.exp(log_w_adjust), max=rb_config['max_adjust_w_clip'])
+            # Manually calculate the new form of the loss
             loss = - torch.mean(w_adjust * log_q_x)
+            # Adjust buffer samples
+            buffer.adjust(log_w_adjust, log_q_x.detach(), indices)
 
     else:
```
Comment (lollcat) [inline examples/aldp.py:497]: On line 509 I found it useful to check that the gradient norm is not `NaN` before doing an optimizer step (this may not be necessary for aldp problem though)
